src/templates/governance-page/ThresholdDaoDataSection.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, Button, HStack, Icon, Stack } from "@chakra-ui/react"
+import { IoDocument } from "react-icons/all"
+import Card from "../../components/Card"
+import { PageSection } from "../../components/PageSection"
+import { H2, H3, LabelMd } from "../../components"
+import ExternalButtonLink from "../../components/Buttons/ExternalButtonLink"
+import { ExternalLinkHref } from "../../components/Navbar/types"
+import { StatBoxGroup } from "../../components/StatBox"
+
+const ThresholdDaoDataSection = () => {
+  const totalTreasuryHoldings = "$50,000,000"
+
+  const stats = [
+    {
+      amount: "50,000,000",
```

Comment:
@r-czajkowski This PR is getting quite large and this data has to be fetched, so if it's OK with you I think we should leave as hard-coded and follow up with another PR

Reply:
Yeep sounds good 👍.